2019-September.html

```diff
@@ -0,0 +1,251 @@
+                    "next-meetup-talk": "My role as a developer",
+                    "next-meetup-speaker": "Henrique Moody"
+                },
+                "images": {
+                    "hoster-small": {
+                        "src": "img/hoster-logos/werkspot.svg",
+                        "alt": "Hosted by Werkspot"
+                    },
+                    "hoster-large": {
+                        "src": "img/hoster-logos/werkspot.svg",
+                        "alt": "Hosted by de Werkspot"
+                    },
+                    "hoster-next": {
+                        "src": "img/hoster-logos/usabilla-large.png",
+                        "alt": "Usabilla"
+                    },
+                    "speaker-face": {
+                        "src": "img/speakers/bob_den_otter.jpeg",
+                        "alt": "Speaker: Bob den Otter"
+                    }
+                }
+            };
+        </script>
+    </head>
+
+    <body>
+
+        <div class="reveal">
+
+            <div class="slides">
+                <section>
+                    <img src="img/logo-transparent-background.png" alt="AmsterdamPHP">
+                    <h4 id="date"></h4>
+
+                    <div class="two-columns" style="margin-top: 4em;">
+                        <div>
+                            <p>Hosted by</p>
+                            <img id="hoster-small" class="hoster" src="" alt=""">
+                        </div>
+
+                        <div>
+                            <ul class="pointless">
+                                <li>18h00 - 18h30: Welcome</li>
+                                <li>18h30 - 19h30: Dinner</li>
+                                <li>19h30 - 20h30: Talk</li>
+                                <li>20h30 - 22h00: Raffle &amp; Social</li>
+                            </ul>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1>Tonight's Sponsors</h1>
+                    <img id="hoster-large" class="hoster big" src="" alt="">
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Tonight's Prizes</h1>
+                </section>
+
+                <section>
+                    <h1>Books!</h1>
+
+                    <img class="hoster big" src="img/hoster-logos/messagebird-full.svg" alt="Sponsor: MessageBird">
+                    <div class="fragment grow hoster-hiring">
+                        We're hiring!
+                    </div>
+                </section>
+
+                <!-- <section data-background="img/pagination_promo.jpg">
+                </section> -->
+
+                <section>
+                    <p>
+                        <img src="img/checkin-to-win.png" alt="Check-in to win"/>
+                    </p>
+
+                    <a href="http://raffles.amsterdamphp.nl">http://raffles.amsterdamphp.nl</a>
```

Comment:
Maybe remove this link since raffler isn't working?

Reply:
Nuked it :fire: